src/test/java/api/CheckoutApi.java

```diff
@@ -5,7 +5,7 @@
     private final String specificPath;
 
     public CheckoutApi() {
-        this.specificPath = "checkoutItem";
+        this.specificPath = "checkout";
```

Comment:
You may as well initialise this inline above, i.e. 

```java
private final String specificPath = "checkout";
```

Then you wouldn't need an explicit constructor.